.github/workflows/build-wheels.yml

```diff
@@ -0,0 +1,81 @@
+name: Build Wheels
+
+# Only run on new tags starting with `v`
+on: [push]
+  # push:
+  #   tags:
+  #     - 'v*'
+
+jobs:
+  build_wheels:
+    name: Build wheel on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [windows-latest, macos-latest]
```

Comment:
Ideally `ubuntu-18.04` (or maybe `ubuntu-latest`) would be included in this list